_quarto.yml

```diff
@@ -14,13 +14,15 @@ website:
         href: get-involved.qmd
       - text: Blog
         menu:
+          - text: All
```

Comment:
Should this be something like "all blog posts" or something? Because it doesn't include presentations, i.e., not all publications.

Reply:
I understand the confusion - I moved the presentations back into the main bar for now and will come with a proposal. May also be helped by the multi-selection you mentioned (which is tracked over in #244).